bin/lint-staged

```diff
@@ -30,4 +38,12 @@ if (cmdline.debug) {
 
 debug('Running `lint-staged@%s`', pkg.version)
 
-require('./src')(console, cmdline.config, !!cmdline.shell, !!cmdline.quiet, !!cmdline.debug)
+lintStaged(console, cmdline.config, !!cmdline.shell, !!cmdline.quiet, !!cmdline.debug).then(
```

Comment:
Would you mind, while you're at it, changing the API of `lintStaged` (`./src/index.js` here) to a configration object instead of multiple arguments? Would make it more intuitive to use in a node context.

Besides, I'm planning on adding a `relative` option soon, so it would be one more argument here.

Maybe if it would work something like this:

```js
lintStaged({ config, shell, quiet, debug }, console)
```

so the second optional argument is the logger.

Reply:
I've changed it to an object. I kept `configPath` so as not to take on the cost/complexity of passing a config object into the function. I shortened the other parameters as you suggested.